content/docs/command-reference/exp/list.md

```diff
@@ -37,7 +37,11 @@ options below).
 - `--rev <commit>` - list experiments derived from the specified Git commit
   (instead of `HEAD`).
 
-- `--all` - list all experiments in the repository (overrides `--rev`).
+- `-n <num>, --num <num>` - list experiments from the `<rev>` commit and `num`
+  commits before it. (`-1` for the whole history of the `<rev>`)
```

Comment:
Same as https://github.com/iterative/dvc.org/pull/3187#discussion_r793240744

Reply:
Looks like this text will have to change again... See https://github.com/iterative/dvc.org/pull/3187#pullrequestreview-883803230.